docs/source/start.rst

```diff
@@ -256,6 +256,12 @@ The :py:class:`~pyquil.quil.Program` object allows us to build up a Quil program
 :py:class:`~pyquil.api.QuantumComputer` object, which specifies what our program should run on (see: :ref:`qvm`). We've also imported all (``*``)
 gates from the ``pyquil.gates`` module, which allows us to add operations to our program (:ref:`basics`).
 
+.. note::
+
+    PyQuil also provides a handy function for you to ensure that a local qvm and quilc are currently running in
+    your environment. To make sure both are available you import `from pyquil.api import local_qvm` and then run
+    `local_qvm()`. This will start a qvm and quilc instances using subprocesses if they have not already been started.
```

Comment:
I would expand the documentation here to include that `local_qvm` can be used in the `with` statement so that the processes are only running during that context. See the docstring https://github.com/rigetti/pyquil/blob/master/pyquil/api/_quantum_computer.py#L631

Maybe we want something more comprehensive than a "note" in the docs.

Reply:
@notmgsk good suggestion. I added in that code snippet to the note.

While this will work for now, I notice that the api module isn't autodoc'd into the hosted documentation. Longer term maybe that could be added in as a page and this would link to the generated version of the docstring.

For now though I think go ahead and merge if you guys approve.